### DISCUSSION TITLE
Oops, something went wrong... invalid_auth - Slack Authentication

### DISCUSSION THREAD

**Discussion by msj321** [requarks/wiki#5124] [Error / Bug Report]
Hi,
I followed the guide in the below link: 
https://docs.requarks.io/auth/slack

When I attempt to login via slack, it requests permission to access our workspace, I allow it and then I get the error page popup stating "Oops, something went wrong... invalid_auth"

Please let me know if there is anything I can do to fix this or if it's something that is currently unavailable.

Thanks

**Comment by Phil-hostwiki** [comment]
Have you checked the logs?

**Reply by msj321** [reply]
Hi Phil, apologies for my ignorance, where can I check the logs?

**Reply by Phil-hostwiki** [reply]
If you are using docker `docker logs container_name`

**Reply by msj321** [reply]
I'm not using Docker, I setup the wiki on a Digital Ocean droplet and followed the guide linked above
